File: TEK3/DOTNET/DOT_AREA_2017/Modules/reaction/system/PrintLine.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Modules.reaction.system
{
    class PrintLine : Reaction
    {
        private Dictionary<String, String> fields = new Dictionary<String, String>();

        public List<String> getFields()
        {
            return (new List<String>() { { "message" } });
        }

        public void setField(Dictionary<String, String> fields)
        {
            this.fields = fields;
        }

        public void execute()
        {
            this.execute(null);
        }

        public void execute(String str)
        {
            if (str != null)
                Console.WriteLine(str);
            else
                Console.WriteLine(fields["message"]);
        }

    }
}

File: TEK3/Java/Java_area_2017/src/main/java/com/area/modules/reaction/system/PrintLine.java
package com.area.modules.reaction.system;

import com.area.modules.Reaction;

import javax.mail.*;
import javax.mail.internet.InternetAddress;
import javax.mail.internet.MimeMessage;
import java.util.*;

public class PrintLine implements Reaction {

    Map<String, String> fields = new HashMap<String, String>();

    public List<String> getFields() {
        return (new ArrayList<String>() {{
            add("message");
        }});
    }
    public void setField(Map<String, String> fields) {
        this.fields = fields;
    }
    public void execute() {
        this.execute(null);
    }
    public void execute(String str) {
        if (str != null)
            System.out.println(str);
        else
            System.out.println(fields.get("message"));
    }
}
